manifests/lib/kube.libsonnet

```diff
@@ -429,6 +429,38 @@
     },
   },
 
+  CronJob(name): $._Object("batch/v1beta1", "CronJob", name) {
+    local cronjob = self,
+ 
+   spec: {
+      jobTemplate: {
+        spec: $.JobSpec {
+          template+: {
+            metadata+: {
+              labels: cronjob.metadata.labels,
+            },
+          },
+        },
+      },
+      schedule: error "Need to provide spec.schedule",
+      successfulJobsHistoryLimit: 10,
+      failedJobsHistoryLimit: 20,
+      concurrencyPolicy: "Forbid",
+    },
+  },
+
+  JobSpec: {
```

Comment:
I think we should reuse this `JobSpec` also for the `Job` object ?

Reply:
Addressed